ota-plus-web/app/reactapp/src/stores/DevicesStore.js

```diff
@@ -192,7 +192,7 @@ export default class DevicesStore {
     resetAsync(this.devicesLoadMoreAsync, true);
     this.devicesCurrentPage++;
     this.devicesOffset += limit;
-    let apiAddress = `${API_DEVICES_SEARCH}?regex=${filter}&limit=${limit}&offset=${this.devicesOffset}`;
```

Comment:
Not sure when this is called, but surely it won't work as expected using `regex` instead of `nameContains`.